doc/Architecture-overview/index.md

```diff
@@ -10,156 +10,157 @@ sort_idx: 15
 * TOC
 {:toc}
 
-First things first, let us take a look at some fundamental concepts necessary for understanding the Kaa architecture and the logical design.
+This section discusses fundamental concepts behind Kaa architecture and logical design.
 
-The Kaa IoT platform consists of the *Kaa server*, *Kaa extensions*, and *endpoint SDKs*.
+[Kaa IoT platform]({{root_url}}Glossary/#kaa-platform) consists of [Kaa server]({{root_url}}Glossary/#kaa-server), [Kaa extensions]({{root_url}}Glossary/#kaa-extensions), and the [endpoint SDKs]({{root_url}}Glossary/#endpoint-sdk).
```

Comment:
[Kaa extensions]({{root_url}}Glossary/#kaa-extensions) term not exist in the [KAA-1160 Glossary update started](https://github.com/kaaproject/kaa/pull/872/) PR.